0chain/system_test
# Free storage tokens test

Resolves https://github.com/0chain/system_test/issues/114

Tests covered
- Create free storage from marker with accounting (currently not working)
- Create free storage with malformed marker should fail
- Create free storage with invalid marker contents should fail
- Create free storage with invalid marker signature should fail
- Create free storage with wrong recipient wallet should fail
- Create free storage with tokens exceeding assigner's individual limit should fail
